perllib/FixMyStreet/App/Controller/Reports.pm

```diff
@@ -180,6 +185,16 @@ sub ward : Path : Args(2) {
             }
             $c->stash->{children} = $children;
         }
+        my $covered = {};
+        $c->cobrand->call_hook('add_parish_wards' => $covered);
+            unless ($covered->{error}) {
+                foreach (values %$covered) {
+                    $_->{url} = $c->uri_for( $c->stash->{body_url}
```

Comment:
These lines have four spaces more indentation than needed. Threw me off a bit when I first read it!